src/lib/common/string.cpp

```diff
@@ -676,3 +676,41 @@ void strReplace(char* to, int toLen, const char* from, const char* oldString, co
 
```

Comment:
Probably in this case, it is better to use several entries, given that the item includes several functional pieces, e.g: NGSI9 basic operation with register/discover and forwarding logic taking into account servicePath for registrations.